database/migrations/2024_09_19_125543_add_default_tab_to_users.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table): void {
+            $table->string('default_tab')->default('following')->after('mail_preference_time');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
```

Comment:
Can you remove this?

Reply:
the docBlock or the entire down function ?

Reply:
We don't have a down method in any migration 👀

So remove the down method.

(I always forget this whole review)

Reply:
thanks.